src/main/resources/static/javascript/mainApp/feedback.controller.js

```diff
@@ -0,0 +1,31 @@
+/* global _, angular, UHGroupingsApp */
+
+(() => {
+
+    /**
+     * This controller contains functions specific to the feedback page.
+     * @param $scope - binding between controller and HTML page
+     * @param $controller - service for instantiating controllers
+     * @param Message - display messages
+     */
+    function FeedbackJsController($scope,$controller, Message) {
+        // regex pattern for email to follow recipient@domain.TLD format
+        $scope.emailPattern = new RegExp("[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\\.[a-zA-Z]{2,}");
+        $scope.feedbackEmail = '';
+
+        angular.extend(this, $controller("GeneralJsController", { $scope }));
+
+        /**
+         * Display a modal when an invalid email is entered on the feedback page
+         */
+        $scope.checkEmail = (event) => {
+            console.log("in check email:" + $scope.feedbackEmail);
+            if ((!($scope.emailPattern.test($scope.feedbackEmail))) && event) {
```

Comment:
Don't forget to remove the `console.log`. Also the outer parenthesis can be removed from 
`!($scope.emailPattern.test($scope.feedbackEmail))`